trajopt/CMakeLists.txt

```diff
@@ -13,8 +13,11 @@ find_package(trajopt_utils REQUIRED)
 find_package(tesseract_common REQUIRED)
 find_package(tesseract_environment REQUIRED)
 find_package(tesseract_visualization REQUIRED)
-find_package(jsoncpp REQUIRED)
+if(NOT TARGET JsonCpp::JsonCpp)
+  find_package(jsoncpp REQUIRED)
+endif()
 find_package(ros_industrial_cmake_boilerplate REQUIRED)
+find_package(osqp REQUIRED)
```

Comment:
Should this be added.

Reply:
Sorry that was from something else and I didn't remove it.